service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -545,31 +545,54 @@ internal class RevurderingServiceImpl(
         fritekst: String,
     ): Either<KunneIkkeForhåndsvarsle, Revurdering> {
         val revurdering = revurderingRepo.hent(revurderingId)
-        if (revurdering?.forhåndsvarsel != null) {
-            return KunneIkkeForhåndsvarsle.AlleredeForhåndsvarslet.left()
+            ?: return KunneIkkeForhåndsvarsle.FantIkkeRevurdering.left()
+
+        fun sendTilAttestering(
```

Comment:
Kan disse inline funksjonene ligge nederst eller må de være definert før man bruker de? Synes den ødelegger lese-flyten litt. Den kalles vel bare en gang, så kan vel evt. droppe funksjonen. Et annen ting er at denne fila begynner å bloate en del. Virker nesten som forhåndsvarsling kunne vært flyttet til en egen service?

Reply:
De må være deklarert før bruk, enig i at de _kan_ være litt hemmende for lesbarhet. Originalt ble den kalt flere enn 1 gang inne i denne blokka, har nok glemt å fjerne den igjen etter at jeg refaktorerte vekk de andre kallene.

Bloat, ja: lar det å evt skille dette ut ligge til en egen jobb.